src/databricks/labs/pytester/fixtures/catalog.py

```diff
@@ -297,8 +297,8 @@ def test_catalog_fixture(make_catalog, make_schema, make_table):
     ```
     """
 
-    def create() -> CatalogInfo:
-        name = f"dummy_C{make_random(4)}".lower()
```

Comment:
Do we prefer the full name or just the name prefix?

Reply:
The convention so far is that when a name is provided it's always the full name instead of just the prefix.